coops/sentinels.tex

```diff
@@ -48,9 +48,9 @@ \subsection*{\MakeUppercase{Map Setup}}
   \item Ignore their yellow borders.
 \end{itemize}
 
-\textbf{4 × Far (II-III) Map tile}
+\textbf{4 × Far (II--III) Map tile}
```

Comment:
All those dashes apply to all scenarios and templates we have here. Does it make sense to update it all?